example/ios/Podfile

```diff
@@ -1,16 +1,41 @@
-require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
-require_relative '../node_modules/react-native/scripts/react_native_pods'
-
-platform :ios, '10.0'
-
-dynamic_frameworks = ['CardVerify']
 
+require_relative '../node_modules/react-native/scripts/react_native_pods'
+require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+platform :ios, '11.2'
```

Comment:
should this be 11.0 with `@available` tags?

Reply:
We can change it to 11.0 since we have all those available annotations in the library now. However, I left it as 11.2 to match the Podfile that has been given to us.